scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/spel/SpelExpression.scala

```diff
@@ -143,7 +143,7 @@ class SpelExpression(
       case literal: LiteralExpression => List(RenderedLiteral(literal.getExpressionString))
       case spelExpr: org.springframework.expression.spel.standard.SpelExpression =>
         // TODO: Should we use the same trick with re-parsing after ClassCastException as we use in ParsedSpelExpression?
-        List(RenderedSubExpression(spelExpr.getValue[String](evaluationContext, classOf[String])))
+        List(RenderedSubExpression(spelExpr.getValue(evaluationContext)))
```

Comment:
let's use `[Any]` explicitly

Reply:
`.getValue` with single parameter does not require generic param